packages/dds/map/src/interfaces.ts

```diff
@@ -233,6 +233,25 @@ export interface IDirectoryEvents extends IEvent {
         target: IEventThisPlaceHolder,
     ) => void);
 
+    /**
+     * Emitted when a subdirectory is undeleted.
```

Comment:
It would probably be helpful to explain this in other words if the reader doesn't understand what "undelete" is. Same for "undispose".
```suggestion
     * Emitted when a previously deleted subdirectory is restored.
```